nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py

```diff
@@ -60,6 +60,9 @@ def __init__(self, cfg: DictConfig, trainer: Trainer):
             seed=self.cfg.get('seed', 1234),
         )
 
+        if not self.cfg.get('fused_bf16'):
+            set_jit_fusion_options()
+
         if isinstance(self.trainer.accelerator_connector.precision_plugin, NLPNativeBfloat16PrecisionPlugin):
```

Comment:
We can remove this?

Reply:
Remove it entirely? I think we want to disable jit fusion for bf16 right?